### PR TITLE
Update uk.m3u

### PR DESCRIPTION
**Pull request by MavijaviGit**

### CODE REVIEW

**Comment by MavijaviGit** [comment]
Added BFBS Channels

File: streams/uk.m3u [hunk 1]
```diff
@@ -135,8 +137,14 @@ https://a.jsrdn.com/broadcast/22680_3BR3zocwi9/-0500/c.m3u8
 #EXTINF:-1 tvg-id="NollyAfrica.uk",Nolly Africa (1080p)
 https://edge-i.evrideo.tv/8808fff1-2262-4c92-8cf4-359adbb692ca_1000026794_HLS/manifest.m3u8
 #EXTINF:-1 tvg-id="NoorTV.uk",Noor TV (480p) [Not 24/7]
```
Comment (Alstruit) [inline streams/uk.m3u:139]:
```suggestion
```

File: streams/uk.m3u [hunk 2]
```diff
@@ -20,6 +20,8 @@ https://liveedge-arisenews.visioncdn.com/live-hls/arisenews/arisenews/arisenews_
 #EXTINF:-1 tvg-id="BBCNewsNorthAmerica.uk",BBC News (North America) (1080p)
 https://d2vnbkvjbims7j.cloudfront.net/containerA/LTN/playlist.m3u8
 #EXTINF:-1 tvg-id="BBCNewsAsiaPacific.uk",BBC News Asia Pacific (720p)
```
Comment (Alstruit) [inline streams/uk.m3u:22]:
```suggestion
#EXTINF:-1 tvg-id="BBCNewsAsiaPacific.uk",BBC News Asia Pacific (720p)
https://cdn4.skygo.mn/live/disk1/BBC_News/HLSv3-FTA/BBC_News.m3u8
```

File: streams/uk.m3u [hunk 3]
```diff
@@ -20,6 +20,8 @@ https://liveedge-arisenews.visioncdn.com/live-hls/arisenews/arisenews/arisenews_
...
+#EXTINF:-1 tvg-id="BBCTwo.uk",BBC Two UK (1080p)
+https://d20opqftbkv2z0.cloudfront.net/live13/bbctwo/bitrate1-clear.isml/live.m3u8
 https://cdn4.skygo.mn/live/disk1/BBC_News/HLSv3-FTA/BBC_News.m3u8
```
Comment (Alstruit) [inline streams/uk.m3u:25]:
```suggestion
```

File: streams/uk.m3u [hunk 4]
```diff
@@ -134,9 +136,14 @@ https://bcovlive-a.akamaihd.net/r2d2c4ca5bf57456fb1d16255c1a535c8/eu-west-1/6058
...
+#EXTINF:-1 tvg-id="NickJr.uk",Nick Jr. UK (1080p)
+https://d20opqftbkv2z0.cloudfront.net/live13/nickjr/bitrate1-clear.isml/live.m3u8
+#EXTINF:-1 tvg-id="Nickelodeon.uk",Nickelodeon UK (1080p)
+https://d20opqftbkv2z0.cloudfront.net/live12/nickelodeon/bitrate1-clear.isml/live.m3u8
+#EXTINF:-1 tvg-id="Nicktoons.uk",NickToons UK (576p)
+https://d20opqftbkv2z0.cloudfront.net/live13/nicktoons/bitrate1-clear.isml/live.m3u8
 #EXTINF:-1 tvg-id="NoorTV.uk",Noor TV (480p) [Not 24/7]
 http://45.76.186.114:8080/hls/noortv.m3u8
-#EXTINF:-1 tvg-id="NoorTV.uk",Noor TV (480p) [Not 24/7]
 https://live.smartmediahouse.tv:8443/hls/ch1.m3u8
```
Comment (Alstruit) [inline streams/uk.m3u:147]:
```suggestion
#EXTINF:-1 tvg-id="NoorTV.uk",Noor TV (480p) [Not 24/7]
https://live.smartmediahouse.tv:8443/hls/ch1.m3u8
```

**Review by BellezaEmporium — COMMENTED** [review]
+ BBC One https://d20opqftbkv2z0.cloudfront.net/live11/bbcone/bitrate1-clear.isml/live.m3u8

though for the BBC channel I believe we have an "uk_bbc" playlist.

**Comment by GiovanniSebastian** [comment]
https://d20opqftbkv2z0.cloudfront.net/live10/cbbc/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live15/bbcthree/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live15/cbeebies/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live18/bbc4/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live17/itv/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live16/itv2/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live17/itvbe/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live14/channel4/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live18/channel5/bitrate1-clear.isml/live.m3u8

**Comment by MavijaviGit** [comment]
> https://d20opqftbkv2z0.cloudfront.net/live10/cbbc/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live15/bbcthree/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live15/cbeebies/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live18/bbc4/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live17/itv/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live16/itv2/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live17/itvbe/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live14/channel4/bitrate1-clear.isml/live.m3u8 https://d20opqftbkv2z0.cloudfront.net/live18/channel5/bitrate1-clear.isml/live.m3u8

Nice alot of channels from BFBS

**Comment by BellezaEmporium** [comment]
https://d20opqftbkv2z0.cloudfront.net/live10/bfbsextra/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live12/bfbssport/bitrate1-clear.isml/live.m3u8
https://d20opqftbkv2z0.cloudfront.net/live13/nepaltv/bitrate1-clear.isml/live.m3u8

**Review by Alstruit — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**